azure-storage/pom.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>spring-cloud-stream-azure-storage-parent</artifactId>
```

Comment:
Not sure what you are doing locally, but this change isn't present in the current PR state.
You should do changes, perform commit and push to `origin`.
The PR here in GH picks up all the changes automatically.